pkg/virtctl/vmexport/vmexport_test.go

```diff
@@ -97,9 +109,10 @@ var _ = Describe("vmexport", func() {
 
 	Context("VMExport fails", func() {
 		It("VirtualMachineExport already exists when using 'create'", func() {
-			vme := utils.VMExportSpecPVC(vmeName, metav1.NamespaceDefault, "test-pvc", secretName)
-			utils.HandleVMExportGet(virtClient, vme, vmeName)
-			err := runCreateCmd(setFlag(vmexport.PVC_FLAG, "test-pvc"))
+			_, err := virtClient.ExportV1beta1().VirtualMachineExports(metav1.NamespaceDefault).Create(context.Background(), vme, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			err = runCreateCmd(setFlag(vmexport.PVC_FLAG, pvcName))
 			Expect(err).To(MatchError(ContainSubstring("VirtualMachineExport 'default/test-vme' already exists")))
```

Comment:
nit: Not related to this PR, but this `ContainSubstring` should be replaced by something else or at least compare the received string against a string from error's constants.

Reply:
What would you like to improve by that?

Reply:
It could be a potential follow up.